shared/Dictionary.lua

```diff
@@ -0,0 +1,51 @@
+Dictionary = class(ValueStorage)
```

Comment:
There's no need to inherit ValueStorage for Dictionary since all we're doing with it is storing some data in this class. You could just have a `self.current_language` variable in this class which would be more readable and simple. We use ValueStorage for in-game entities for easily setting and getting data on them.